src/app/Dashboard/AutomatedAnalysis/AutomatedAnalysisCard.tsx

```diff
@@ -636,28 +799,37 @@ export const AutomatedAnalysisCard: React.FC<AutomatedAnalysisCardProps> = (prop
                 isSmall
                 isDisabled={isLoading}
                 isAriaDisabled={isLoading}
-                aria-label="Delete automated analysis"
+                aria-label={t('AutomatedAnalysisCard.ARIA_LABELS.DELETE_AA')}
                 onClick={clearAnalysis}
                 variant="control"
                 icon={<TrashIcon />}
               />
             </ToolbarItem>
             <ToolbarItem>
               <Checkbox
-                label="Show N/A scores"
+                label={t('AutomatedAnalysisCard.CHECKBOX_LABEL_SHOW_NA_SCORES')}
                 isChecked={showNAScores}
                 onChange={handleNAScoreChange}
                 id="show-na-scores"
                 name="show-na-scores"
               />
             </ToolbarItem>
+            <ToolbarItem>
+              <Switch
+                label={t('AutomatedAnalysisCard.SWITCH_LABEL_LIST_VIEW')}
+                isChecked={showListView}
+                onChange={() => setShowListView(!showListView)}
```

Comment:
Access to current state should use factory function syntax? `setState((state) => !state)`;